client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -79,14 +79,15 @@ export default function SitesDashboard() {
 		} );
 	}, [ sitesViewState.filters, setAgencyDashboardFilter, showOnlyFavorites ] );
 
-	const { data, isError, isLoading, refetch } = useFetchDashboardSites(
-		isPartnerOAuthTokenLoaded,
-		sitesViewState.search,
-		sitesViewState.page,
-		agencyDashboardFilter,
+	const { data, isError, isLoading, refetch } = useFetchDashboardSites( {
+		isPartnerOAuthTokenLoaded: false,
```

Comment:
Should we hardcode the `false` here?

Reply:
I apologize If I didn't get your question right. Do you mean you want `isPartnerOAuthTokenLoaded` to be optional?

Reply:
If I'm not mistaken, previously we had:

```
useFetchDashboardSites( isPartnerOAuthTokenLoaded, ...)
```

And we've changed this to:

```
useFetchDashboardSites( {
		isPartnerOAuthTokenLoaded: false,
		...
});
```

So I'm just curious if it's okay to hardcode `false` like this?

Reply:
I believe this was setup to wait for the Partner key to be loaded in the UI before retrieving the sites. Now that we don't rely on w/ partner keys in A4A, we can set this to false as passing agencyId will enable the query.

Reply:
That makes sense! Let's 🚢  it then.